targets/tizenrt-artik053/apps/jerryscript/jerry_main.c

```diff
@@ -155,40 +154,46 @@ print_unhandled_exception (jerry_value_t error_value, /**< error value */
     {
       unsigned int err_line = 0;
       unsigned int err_col = 0;
+      char *path_str_p = NULL;
+      size_t path_str_end = 0;
 
       /* 1. parse column and line information */
-      for (jerry_size_t i = 0; i < sz; i++)
+      for (uint32_t i = 0; i < sz; i++)
       {
-        if (!strncmp ((char *) (err_str_buf + i), "[line: ", 7))
+        if (err_str_buf[i] == '[')
         {
-          i += 7;
+          i++;
 
-          char num_str[8];
-          unsigned int j = 0;
+          if (err_str_buf[i] == '<')
+          {
+            break;
+          }
 
-          while (i < sz && err_str_buf[i] != ',')
+          path_str_p = (char *) (err_str_buf + i);
+          while (i < sz && err_str_buf[i] != ':')
           {
-            num_str[j] = (char) err_str_buf[i];
-            j++;
             i++;
           }
-          num_str[j] = '\0';
 
-          err_line = (unsigned int) strtol (num_str, NULL, 10);
+          path_str_end = i++;
 
-          if (strncmp ((char *) (err_str_buf + i), ", column: ", 10))
+          char num_str[8];
+          unsigned int j = 0;
+
+          while (i < sz && err_str_buf[i] != ':')
           {
-            break; /* wrong position info format */
+            num_str[j++] = (char) err_str_buf[i++];
           }
+          num_str[j] = '\0';
 
-          i += 10;
+          err_line = (unsigned int) strtol (num_str, NULL, 10);
```

Comment:
As the strtol is used, we could leverage it to in every case where the str->number conversion is required. With it we would not need to manually copy the chars into the num_str array, just point the start of the string where the number should be and it will return the the end pointer where the value is not convertible. This would reduce a bit of code here.